challenges/README.md

```diff
@@ -13,7 +13,7 @@ same as the users object on Users Api
 | :--------------------------------: | :-----------------------------: |
 |  [GET /challenges](#get-challenges)| Returns all challenges          |
 |[POST /challenges](#post-challenges)|       Creates new challenge     |
-|[POST /challenge/subscribe](#post-challengesubscribe) | subscribes users to challenge|
+|[POST /challenge/subscribe](#post-challengessubscribe) | subscribes users to challenge|
 
```

Comment:
While you are at it, can we please format this table too using [this](https://marketplace.visualstudio.com/items?itemName=fcrespo82.markdown-table-formatter) or some similar tool?

Reply:
Changes commited